### PR TITLE
Install & configure openshift data foundation including argocd app

### PR DESCRIPTION
**Pull request by rbo**
Part of #1471

### CODE REVIEW

File: argocd/overlays/moc-infra/applications/envs/emea/morty/cluster-management/odf.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: odf
+spec:
+  destination:
+    name: morty
+    namespace: argocd
```
Comment (HumairAK) [inline argocd/overlays/moc-infra/applications/envs/emea/morty/cluster-management/odf.yaml:9]: did you mean `namespace: openshift-storage` ?

File: argocd/overlays/moc-infra/applications/envs/emea/morty/cluster-management/local-storage.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: local-storage
+spec:
+  destination:
+    name: morty
+    namespace: argocd
```
Comment (HumairAK) [inline argocd/overlays/moc-infra/applications/envs/emea/morty/cluster-management/local-storage.yaml:9]: did you mean: `namespace: openshift-local-storage` ?

File: odf/base/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: openshift-storage
+
+resources:
```
Comment (HumairAK) [inline odf/base/kustomization.yaml:7]: you can remove this line entirely

File: argocd/overlays/moc-infra/applications/envs/emea/morty/cluster-management/odf.yaml (hunk 1 above)
Comment (rbo) [reply argocd/overlays/moc-infra/applications/envs/emea/morty/cluster-management/odf.yaml:9]: Indeed, fixed. Thanks!

File: argocd/overlays/moc-infra/applications/envs/emea/morty/cluster-management/local-storage.yaml (hunk 2 above)
Comment (rbo) [reply argocd/overlays/moc-infra/applications/envs/emea/morty/cluster-management/local-storage.yaml:9]: Indeed, fixed. Thanks!

**Comment by HumairAK** [comment]
Bah should have squash rebased. Oh well.

**Comment by HumairAK** [comment]
https://argocd.operate-first.cloud/applications/odf-morty

https://argocd.operate-first.cloud/applications/local-storage-morty

**Comment by rbo** [comment]
Just for documentation: part of  https://github.com/operate-first/apps/issues/1471

**Pull request merged**